database/src/main/kotlin/no/nav/su/se/bakover/database/vedtak/VedtakPostgresRepo.kt

```diff
@@ -540,6 +564,17 @@ private fun lagreKlagevedtakKnytningTilRevurdering(vedtak: Stønadsvedtak, sessi
     )
 }
 
+private fun lagreKlagevedtakKnytningTilRegulering(vedtak: Stønadsvedtak, session: Session) {
+    lagreVedtaksknytning(
+        behandlingVedtakKnytning = BehandlingVedtakKnytning.ForRegulering(
+            vedtakId = vedtak.id,
+            sakId = vedtak.behandling.sakId,
+            reguleringId = vedtak.behandling.id,
+        ),
+        session = session,
+    )
+}
+
 private fun lagreKlagevedtakKnytningTilSøknadsbehandling(vedtak: Stønadsvedtak, session: Session) {
     lagreVedtaksknytning(
```

Comment:
En filleting, men man kan kanskje bestemme seg for om det heter `vedtaksknytning` eller `vedtakKnytning`.